worker/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"cloud.google.com/go/pubsub"
+	"context"
+	log "github.com/sirupsen/logrus"
+	"os"
+	"strconv"
+)
+
+type CSVWorker struct {
+	sample []byte
+}
+
+func init() {
+	// Only log the warning severity or above.
+	if "DEBUG" == getEnv("LOG_LEVEL", "DEBUG") {
```

Comment:
Is the plan to offer all the various logging levels going forward? It seems odd to hard-code it to just DEBUG or INFO, even if those are the ones we're most likely to use

Reply:
I wasn't a fan of this either, but couldn't see from the library how to do it. There must be a way though, I'll look into it.